pymongo/helpers.py

```diff
@@ -90,6 +90,9 @@
 # Server code raised when re-authentication is required
 _REAUTHENTICATION_REQUIRED_CODE: int = 391
 
+# Server code raised when authentication fails.
+_AUTHENTICATION_FAILURE_CODE: int = 18
```

Comment:
Use this constant instead of the magic number `18` in `auth_oidc`?

Reply:
Done